### DISCUSSION TITLE
How do I modify the default collation configuration and have it take effect?

### DISCUSSION THREAD

**Discussion by dongluo-smile** [apache/doris#40873] [A - General / Q&A]
Through the SHOW CHARACTER SET; Command view display
| Charset | Description | Default collation | Maxlen
| ----- | ----- | ----- | ----- |
| utf8mb4 | UTF-8 Unicode | utf8mb4_0900_bin | 4 |

SHOW COLLATION; Command view display
| Collation | Charset | Id | Default | Compiled | Sortlen
| ----- | ----- | ----- | ----- | ----- | ----- |
| utf8mb4_0900_bin | utf8mb4 | 309 | Yes | Yes | 1 |
| utf8mb3_general_ci | utf8mb3 | 33 | Yes | Yes | 1 |

**Problem: utf8mb4_0900_bin Collation rules cause case sensitivity when querying data, such as（'ABC' != 'abc'）**

**Expect: Change the default collation to utf8mb4_general_ci, query data is case insensitive, for example（'ABC' == 'abc'）**

**Modify the fe.conf and be.conf configuration files to add default_character_set = utf8mb4
default_collation = utf8mb4_general_ci And restart, found no effect.**

**May I ask you, how to configure it to make it effective?**

**Comment by ixzc** [comment]
建议英文提问，如使用中文提问，可以去ask.selectdb.com中文技术论坛。

**Reply by dongluo-smile** [reply]
好的，谢谢，已转英文。
